clarify that AS that only supports pre-auth grant can omit response_types_supported metadata (#249)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

      * `background_color`: OPTIONAL. String value of a background color of the Credential represented as numerical color values defined in CSS Color Module Level 37 [@!CSS-Color].
      * `text_color`: OPTIONAL. String value of a text color of the Credential represented as numerical color values defined in CSS Color Module Level 37 [@!CSS-Color].

> Note: An Authorization Server that only supports Pre-Authorized Code grant type MUST omit `response_types_supported` parameter in its metadata despite [@!RFC8414] mandating it.

Reviewer: ```suggestion
Note: An Authorization Server that only supports the Pre-Authorized Code grant type can omit `response_types_supported` parameter in its metadata despite [@!RFC8414] mandating it.
```

Author: ```suggestion
An Authorization Server that only supports the Pre-Authorized Code grant type MAY omit the `response_types_supported` parameter in its metadata despite [@!RFC8414] mandating it.
```
